src/routes/Login.svelte

```diff
@@ -68,36 +157,81 @@
+        </h1>
+        <h1>{I18n.t('login.student.subtitle')}</h1>
+        <img
+          src="https://via.placeholder.com/200?text=Placeholder"
+          alt="student login" />
+        <p class="action">{I18n.t('login.student.action')}</p>
+        <LoginButton
+          label={I18n.t('login.student.button')}
+          onClick={() => logIn(role.STUDENT)} />
+      </div>
+      <div class="login-card" class:none={visible}>
+        <div class="titleAndBackButton">
+          <span class="navigateBackButton" on:click={() => toggleLoginCreateAccount()}>{@html navigateBack}</span>
+          <span>
+            <h1 class="bold">
+              {@html I18n.t('login.student.accountCreation.title')}
+            </h1>
+          </span>
+        </div>
+        <h1>{I18n.t('login.student.accountCreation.subtitle')}</h1>
+        <p class="text-align-left">
+          {I18n.t('login.student.accountCreation.require')}
+        </p>
+        <div class="account-creation-steps">
+          <AccountCreationSteps activeStep={1} />
+        </div>
+        <div class="create-account-button">
+          <Button
+            onClick={() => alert('make account')}
+            label={I18n.t('login.student.accountCreation.step1')} />
+        </div>
+      </div>
+      <div class="no-account" class:no-account-active={!visible}>
+        <div class="account-creation">
+          <p>{@html visible ? I18n.t('login.student.accountCreation.askAccountNo') : I18n.t('login.student.accountCreation.askAccountYes') }</p>
+        </div>
+        <div class="account-creation">
+          <p class="styleAsLink" on:click={() => toggleLoginCreateAccount()}>
+            {@html visible ? I18n.t('login.student.accountCreation.startAccount') : I18n.t('login.student.accountCreation.logInAccount')}
+          </p>
         </div>
-        <span>
-          {@html I18n.t('login.student.edu_id_info', {
-            url: 'https://www.surf.nl/'
-          })}
-        </span>
       </div>
     </div>
 
-    <div class="login-card">
-      <h1>{I18n.t('login.teacher.title')}</h1>
-      <img
-        src="https://via.placeholder.com/200?text=Placeholder"
-        alt="teacher login" />
-      <LoginButton
-        label={I18n.t('login.teacher.button')}
-        className="grey"
-        onClick={() => logIn(role.TEACHER)} />
+    <div class="login-element">
+      <div class="login-card {visible ? '' : 'overlay small-screen-none'}">
+        <h1 class="bold">
+          {@html I18n.t('login.teacher.title')}
+        </h1>
+        <h1>{I18n.t('login.teacher.subtitle')}</h1>
+        <img
+          src="https://via.placeholder.com/200?text=Placeholder"
+          alt="teacher login" />
+        <p class="action">{I18n.t('login.teacher.action')}</p>
+        <LoginButton
+          label={I18n.t('login.teacher.button')}
+          onClick={() => logIn(role.TEACHER)} />
+      </div>
+
+      <div class="no-account {visible ? '' : 'hidden small-screen-none'}">
```

Comment:
You can use the shorthand here as well

Reply:
Svelte doesn't support shorthand for two classes. Do you prefer to see the shorthand twice over the JS ternary operator?

Reply:
Sorry, didn't spot that there were two. I think that the shorthand is still easier to read, but that's a matter of personal preference so no matter!